Sources/Orbit/Components/AlertInline.swift

```diff
@@ -0,0 +1,216 @@
+import SwiftUI
+
+/// Breaks the main user flow to present information.
+///
+/// There are times when just simple information isn’t enough and the user needs
+/// to take additional steps to resolve the problem or get additional details.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/alert/)
+/// - Important: Component expands horizontally unless prevented by `fixedSize` or `idealSize` modifier.
+public struct AlertInline<Icon: View, Button: View>: View {
+
+    let title: String
```

Comment:
We can start adding `private` to all properties to avoid accidental usage of synthesised inits in previews.

Reply:
Added.